core/sr-arithmetic/src/biguint.rs

```diff
@@ -0,0 +1,825 @@
+// Copyright 2019 Parity Technologies (UK) Ltd.
```

Comment:
Has anything significant changed here? 

- If so, would be nice to highlight them since this PR is meant to just re-locate files + fuzz them. Both of which should not need any significant change to the lib itself. 
- If not but you intend to so, I'd be happy to do that in a separate PR. I really find it bad review-wise when a PR that re-locates stuff also changes them.

Reply:
No, I haven't significantly changed any internal logic

Reply:
Great, let's keep it that way for the reason mentioned and I'll be happy to work on any sort of optimisation of you find suitable, in a separate PR.